.gitignore

```diff
@@ -24,7 +24,7 @@ firebase-debug.log
 __pycache__/
 
 cloud/gcp/terraform.tfvars
-cloud/gcp/udmi-sites.tf
+cloud/gcp/udmi-site.tf
```

Comment:
This file is meant to have one (minimum) or more sites, hence the `udmi-sites` name.
I think it's more appropriate keeping the name as `udmi-sites.tf`